models/UserManager.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+class UserManager extends Model
+{
+    // Fonction permettant de vérifier que la session est bien active sous le role administrateur
+    function admin(){
+        if(isset($_SESSION['admin'])){
+            $tableau = [
+                'email'     =>  $_SESSION['admin'],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_SESSION Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)